pom.xml

```diff
@@ -211,10 +218,18 @@
                 <version>2.13.3</version>
             </dependency>
 
+<!--            <dependency>-->
+<!--                <groupId>org.slf4j</groupId>-->
+<!--                <artifactId>slf4j-log4j12</artifactId>-->
+<!--                <version>1.7.16</version>-->
+<!--                <scope>runtime</scope>-->
+<!--            </dependency>-->
+
+            <!-- log4j2 dependency for liquibase -->
             <dependency>
-                <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-log4j12</artifactId>
-                <version>1.7.16</version>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-slf4j-impl</artifactId>
+                <version>2.14.1</version>
```

Comment:
I would suggest to stick with the 2.13.3 version, for consistency reasons.

Reply:
Changed the version as suggested